pyproject.toml

```diff
@@ -32,6 +32,7 @@ extend-select =[
     "RSE",
     "RET",
     "SLF",
+    "FURB",
```

Comment:
This is actually a long list of changes: https://docs.astral.sh/ruff/rules/#pyupgrade-up

I guess I'm good with it but I only skimmed though the list.